src/components/Header/Header.tsx

```diff
@@ -50,6 +51,7 @@ const Header = ({ nav, user }: Props) => {
           </NextLink>
         )}
       </div>
+
       {nav !== undefined && (
```

Comment:
Revert alll changes in this file. You just reordered things. Should not be any changes here.